Update ClamAV engine version together with GitLab (#4582) (#4603)
DataBiosphere/azul: OPERATOR.rst

label from that PR. Remove the ``base`` label from the blocking PR. Lastly, remove the blocking
relationship.

> Upgrading GitLab + ClamAV

Author: ```suggestion
Upgrading GitLab & ClamAV
```

^^^^^^^^^^^^^^^^^^^^^^^^^

Operators must check for updates to GitLab and ClamAV on a monthly basis in
addition to triaging GitLab security releases that occur during the month.
An email notification is sent to ``azul-group@ucsc.edu`` when a GitLab security
release is available. Discuss with the lead the **Table of Fixes** referenced in

> the release blog post to determine the urgency of the update. Email
> notifications should also be received when ClamAV releases become available.

Author: ```suggestion
the release blog post to determine the urgency of the update. An email
notification should also be received when ClamAV releases become available.
```

The current version of GitLab installed can be found on the ``/help`` endpoint
of `GitLab dev`_, and the available releases can be found on the
`GitLab Docker image`_ page. When updating the GitLab instance, check if there
are applicable updates to the `GitLab runner image`_. Use the latest runner
image whose major and minor version match that of the GitLab image. Similarly,
check for available releases to ClamAV in the `ClamAV image`_. The current
version of ClamAV image being used can be found by running::

    cat $project_root/terraform/gitlab/gitlab.tf.json.template.py | grep 'clamav_image ='

> Before starting the update process, check the `GitLab release notes`_ upgrading

Author: ```suggestion
Before starting the update process, check the `GitLab release notes`_ for upgrading
```

> Before starting the update process, check the `GitLab release notes`_ upgrading
> instructions. When upgrading across multiple GitLab versions, follow the

Author: ```suggestion
Before starting the update process, check the `GitLab release notes`_ and `ClamAV release notes`_ for any upgrading instructions. When upgrading across multiple GitLab versions, follow the
```
